src/documentation/chunk.rs

```diff
@@ -107,80 +107,92 @@ impl CheckableChunk {
             &range
         );
 
-        let Range { start, end } = range.clone();
+        let Range { start, end } = range;
         let mut active = false;
         self.source_mapping
             .iter()
-            .skip_while(|(sub, _span)| sub.end <= start)
-            .take_while(|(sub, _span)| end <= sub.end)
+            .skip_while(|(fragment_range, _span)| fragment_range.end <= start)
+            .take_while(|(fragment_range, _span)| end <= fragment_range.end)
```

Comment:
I like the new variable names :+1:

Reply:
what does `fragment_range` means here? if I may ask

Reply:
I think it's best explained in the https://github.com/drahnr/cargo-spellcheck/pull/74/files/27c8c0f661e02c310dfa14a9d6475b73f1312cd2#diff-5fe541b5263f70cb56752ad21183cdddR1-R13 if not that must be expanded.